LoginScreenLayout/Sources/AuthViewController.swift

```diff
@@ -1,10 +1,348 @@
 import UIKit
 
 class AuthViewController: UIViewController {
+    // MARK: - Properties
+
+    // Screen dimensions
+    private lazy var mainScreenBounds = UIScreen.main.bounds
+    private lazy var screenWidth = mainScreenBounds.width
+    private lazy var screenHeight = mainScreenBounds.height
+
+    // Ratios for layout constraints
+    private lazy var backgroundImageHeightRatio: CGFloat = 0.65
+    private lazy var centerYOffsetTitleLabelRatio: CGFloat = -0.35
+    private lazy var topMarginLoginTextFieldRatio: CGFloat = 0.045
+    private lazy var leadingTrailingMarginRatio: CGFloat = 0.18
+    private lazy var heightTextFieldRatio: CGFloat = 0.05
+    private lazy var topMarginPasswordTextFieldRatio: CGFloat = 0.022
+    private lazy var topMarginLoginButtonRation: CGFloat = 0.06
+    private lazy var heightButtonRatio: CGFloat = 0.045
+    private lazy var topMarginButtonForgotRatio: CGFloat = 0.03
+    private lazy var topMarginOrLabelRatio: CGFloat = 0.225
+    private lazy var topMarginDividerOrRatio: CGFloat = 0.5
+    private lazy var leadingTralingMarginDividerRatio: CGFloat = 0.03
+    private lazy var heightDividerRatio: CGFloat = 1
+    private lazy var topMarginAuthButtonRatio: CGFloat = 0.03
+    private lazy var widthAuthButtonRation: CGFloat = 0.855
+    private lazy var topMarginSingUpRatio: CGFloat = 0.08
+    private lazy var cornerRadiusTextFieldRatio: CGFloat = 0.025
+    private lazy var cornerRadiusButtonRatio: CGFloat = 0.0225
+
+    // MARK: - UI Elements
+
+    private lazy var backgroundImageView: UIImageView = {
+        let image = UIImage(named: "backgroundLoginScreen")
+        let imageView = UIImageView(image: image)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleToFill
+        return imageView
+    }()
+
+    private lazy var loginTitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Login"
+        label.textColor = .white
+        label.font = UIFont.systemFont(ofSize: 24, weight: .bold)
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var loginTextField: UITextField = {
+        let textField = UITextField()
+        let loginIcon = UIImage(named: "loginIcon")
+        let checkIcon = UIImage(named: "checkIcon")
+
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        textField.setLeftIcon(loginIcon!)
+        textField.setRightIcon(checkIcon!)
+        textField.backgroundColor = .white
+        textField.layer.cornerRadius = cornerRadiusTextFieldRatio * screenHeight
```

Comment:
```suggestion
        textField.layer.cornerRadius = Metric.cornerRadiusTextFieldRatio * Metric.screenHeight
```
пример как можно обратиться будет к ним